.github/workflows/ci.yml

```diff
@@ -0,0 +1,39 @@
+name: CI
+
+on:
+  push:
+    branches: [ '*' ]
+
+jobs:
+  ci-rails-app:
```

Comment:
this is a sinatra app at present, but maybe it will grow up to be a rails app someday